crypt-gensalt-static.c

```diff
@@ -15,20 +15,29 @@
    <https://www.gnu.org/licenses/>.  */
 
 #include "crypt-port.h"
+#include <stdlib.h>
 
 /* The functions that use global state objects are isolated in their
    own files so that a statically-linked program that doesn't use them
    will not have the state objects in its data segment.  */
 
 #if INCLUDE_crypt_gensalt
+
+static THREAD_LOCAL char *obuf;
```

Comment:
Shouldn't we explicitly initialize the pointer with a `NULL` value?

Reply:
Technically not necessary (static pointers are auto-initialized to NULL, thread-local or not) but I suppose it would be clearer.